speechbrain/lobes/models/huggingface_wav2vec.py

```diff
@@ -0,0 +1,95 @@
+"""This lobes enables the integration of huggingface pretrained wav2vec2 models.
+
+Reference: https://arxiv.org/abs/2006.11477
+Reference: https://arxiv.org/abs/1904.05862
+Transformer from HuggingFace needs to be installed:
+https://huggingface.co/transformers/installation.html
+
+Authors
+ * Titouan Parcollet 2021
+"""
+
+import torch
+import torch.nn.functional as F
+from torch import nn
+
+# We check if transformers is installed.
+try:
+    from transformers import Wav2Vec2Model
+except ImportError:
+    print("Please install transformer from HuggingFace to use wav2vec2!")
+
+
+class HuggingFaceWav2Vec2(nn.Module):
+    """This lobes enables the integration of HuggingFace
+    pretrained wav2vec2.0 models.
+
+    Source paper: https://arxiv.org/abs/2006.11477
+    Transformer from HuggingFace needs to be installed:
+    https://huggingface.co/transformers/installation.html
+
+    The model can be used as a fixed features extractor or can be finetuned. It
+    will download automatically the model from HuggingFace.
+
+    Arguments
+    ---------
+    source : str
+        HuggingFace hub name: e.g "facebook/wav2vec2-large-lv60"
+    save_path : str
+        Path (dir) of the downloaded model.
+    output_norm : bool (default: True)
+        If True, a layer_norm (affine) will be applied to the output obtained
+        from the wav2vec model.
+    freeze : bool (default: True)
+        If True, the model is frozen. If False, the model will be trained
+        alongside with the rest of the pipeline.
+
+    Example
+    -------
+    >>> inputs = torch.rand([10, 600])
+    >>> model_hub = "facebook/wav2vec2-base-960h"
+    >>> save_path = "savedir"
+    >>> model = HuggingFaceWav2Vec2(model_hub, save_path)
+    >>> outputs = model(inputs)
+    >>> outputs.shape
+    torch.Size([10, 1,  768])
+    """
+
+    def __init__(self, source, save_path, output_norm=True, freeze=True):
+        super().__init__()
+
+        # Download the model from HuggingFace and load it.
+        self.model = Wav2Vec2Model.from_pretrained(source, cache_dir=save_path)
```

Comment:
If the save_path/cache_dir changes and the model was already cached this line takes endless time on my machine
After removing the cache_dir argument it loaded within some seconds

Reply:
Really. ? interesting, I don't see any reason why this would happen. Hum.

Reply:
For an unknown reason, also models obtained from HF seem to not converge when fine-tuned. While Fairseq ones work really well. I need to investigate this.

Reply:
Ok, it was just a normalization issue eheh

Reply:
Did a quick run with commonvoice recipe, transducer
Just replaced the model with wave2vec2, getting mismatch error, I think I have to learn more about your design how to configure it correctly. 
Looking forward to your recipe :)

Reply:
I can share with you my TIMIT recipe if you wish.

Reply:
Would be cool 😎